Run Travis for PHP 7.4 (Moodle 3.8), too.

lucaboesch opened the pull request:
We should let it check for PHP 7.4, too.

lucaboesch committed:
Run Travis for PHP 7.4 (Moodle 3.8), too.

dmitriim commented:
It fails in PHP 7.4 Seems like https://github.com/TECLIB/CFPropertyList/pull/61 needs to be applied to plist. As done for core see https://tracker.moodle.org/browse/MDL-68589

lucaboesch committed:
MDL-68589 plist: PHP 7.4 curly bracket fixes

Note this is, partially, TECLIB/CFPropertyList#61

I only have applied the stricly required PHP 7.4 changes, keeping
apart the intval() ==> (int) change, because that's unrelated to
this fix.

lucaboesch commented:
Dmitrii, I amended the patch.
Best,
Luca

dmitriim commented:
Thanks @lucaboesch Looks good. Will merge once ci is green.
